### ISSUE TITLE
[FEATURE] Pro tip for mod-related crashes

### ISSUE THREAD

**Issue by KendrickLamar2003** [Cartrigger/QCXRsoonbot#54]
### 📂 What category is this feature in?

Information

### 📝 Description of Feature

This has been retrieved from my 3rd major README overhaul PR, and it's extremely useful and can help users fix crashes by themselves! Rework and rephrase this to your heart's content, but please remember to credit!

Did you experience a mod-related crash when loading MC? Look through latestlog.txt on Android/data/com.qcxr.qcxr (through SideQuest), scroll to the bottom, and you should see a Fabric error, which says what mods are causing the crash, and fix it!

**Comment by swaggio7** [comment]
This comes from many months of testing and removing mods one by one when I can just look through the logs.

**Comment by swaggio7** [comment]
HOLY SHIT GITHUB

**Comment by swaggio7** [comment]
Didnt mean to do that

**Comment by swaggio7** [comment]
shit, this is already in the bot.
